tests/unit_tests/test_wave_06.py

```diff
@@ -228,7 +238,10 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert not result
+    assert tai.inventory == [item_a, item_b, item_c]
+    assert jesse.inventory == [item_d, item_e, item_f]
```

Comment:
I'm a fan of checking to see if these inventories are exactly what we are expecting. As developers, we get to dictate how different situations should work which means we know exactly what the output should be. Tests like this also give other developers a good idea of how items should move between inventories if at all!